app/lib/redis_configuration.rb

```diff
@@ -39,7 +39,38 @@ def namespace
   end
 
   def url
-    ENV['REDIS_URL']
+    if ENV['REDIS_SENTINEL']
```

Comment:
You could use your `#sentinel_mode?` method here to make this look a bit nicer:

```suggestion
    if sentinel_mode?
```